src/main/resources/roleconfig/role_common.json

```diff
@@ -1156,6 +1192,86 @@
       }
     ]
   },
+  {
+    "name": "challenged-access-judiciary",
```

Comment:
no need to use another role config, use existing specific-access-judiciary and add jurisdiction in attributes. Don't worry about caseType